internal/env/config.go

```diff
@@ -51,7 +51,7 @@ type Config struct {
 	AtomicUpgrade      bool     `split_words:"true"`                 // Pass --atomic to `helm upgrade`
 	CleanupOnFail      bool     `envconfig:"cleanup_failed_upgrade"` // Pass --cleanup-on-fail to `helm upgrade`
 	LintStrictly       bool     `split_words:"true"`                 // Pass --strict to `helm lint`
-	EnableV2Conversion bool     `split_words:"true"`                 // Whether or not to use 2to3 convert to migrate Releases from v2 to v3
+	EnableV2Conversion bool     `split_words:"true" default:"true"`  // Whether or not to use 2to3 convert to migrate Releases from v2 to v3
```

Comment:
An option that starts with "Enable*" that defaults to true is kind of weird. This should probably be `DisableV2Conversion` and use the default `false` value..

Reply:
yep, makes total sense. Changed the logic.